### ISSUE TITLE
loss weight decay of self defined

### ISSUE THREAD

**Issue by jjjcs** [open-mmlab/mmsegmentation#925]
Hi,
im now using a self defined loss, and want to realize following function.
alpha = 1
for e in epochs:
   loss = loss1 + (1 - alpha)*loss2
   alpha -= 0.01

how can i call the number of epoch in my loss?

**Comment by MengzhangLI** [comment]
Hi, to add new loss, you can check out the docs below:

https://github.com/open-mmlab/mmsegmentation/blob/master/docs/tutorials/customize_models.md

Hoping it is useful to you and look forward to your reply.
